src/build-mounting-container.ts

```diff
@@ -0,0 +1,40 @@
+import { ElementOrSelector } from './types';
+
+/**
+ * Helper function for getting an element when given a string or element
+ * @param target a selector or the container
+ * @returns the container that we can mount on
+ */
+const getContainer = (target: ElementOrSelector): HTMLElement => {
```

Comment:
DocStrings no longer have types in them. These _should_ be inherited from the Types in the method definition / implementation.